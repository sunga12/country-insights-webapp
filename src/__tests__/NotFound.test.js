import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router';
import Store from '../redux/store';
import NotFound from '../components/NotFound';

it('MyProfile component test', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Provider store={Store}>
          <NotFound />
        </Provider>
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
