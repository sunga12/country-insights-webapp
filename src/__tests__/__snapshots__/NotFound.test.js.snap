// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyProfile component test 1`] = `
<h1>
  Not Found
</h1>
`;
